src/libraries/System.IO.FileSystem/tests/Base/BaseGetSetTimes.cs

```diff
@@ -37,7 +37,7 @@ protected T CreateSymlinkToItem(T item)
 
         protected abstract string GetItemPath(T item);
 
-        public abstract IEnumerable<TimeFunction> TimeFunctions(bool requiresRoundtripping = false);
+        public abstract IEnumerable<TimeFunction> TimeFunctions(bool requiresRoundtripping = false, bool testHandles = true);
```

Comment:
Can you add a comment that explains what `testHandles` and also `requiresRoundtripping` means?